src/directives/chat/chat-panel/chat-panel.directive.scss

```diff
@@ -11,7 +11,7 @@
   height: 100%;
 
   .chat-panel-content {
-    max-width: 50vw;
+    max-width: 96%;
```

Comment:
Isn't this contradicting `width: 100%` ?

Reply:
Not sure, let me check.

Reply:
Yeah, it was redundant. I've removed it and adjusted the `width` instead.